src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    public void addingTasks(){
```

Comment:
Maybe can change the method's name to be a more descriptive test name?

For example, you can rename it to be something like `addTasks_validInput_getCorrectToString`.